app/frontend/src/components/base/BaseTime.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="d-flex flex-row">
+    <div class="d-flex flex-row">
+      <div>{{ minutes }}</div>
+      <div class="ml-1" :lang="lang" :class="{ 'dir-rtl': isRTL }">
+        {{ $t('trans.baseTime.minutes') }}
+      </div>
+    </div>
+    <div class="d-flex flex-row ml-2">
+      <div>{{ seconds }}</div>
+      <div class="ml-1" :lang="lang" :class="{ 'dir-rtl': isRTL }">
+        {{ $t('trans.baseTime.seconds') }}
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import moment from 'moment';
+import { mapGetters } from 'vuex';
+
+export default {
+  name: 'BaseTime',
+  props: {
+    action: { type: String, default: '' },
+  },
+  data() {
+    return {
+      timerDate: moment().add(15, 'minutes'),
```

Comment:
This 15 minutes value should be based on the token - it could be different in different environments or deployments, and we might wand to change it in Keycloak, which would break it here.

Reply:
The token expires every 5 minutes, and the refresh token every 30 minutes. Addition of both the Idle time and timer counter down is 25 minutes. This would allow users to refresh the token if they stay signed in

Reply:
But if we go into Keycloak and change the token to expire every 60 minutes, then this will be displayed too early. I think all the logic should be based on the timeout value in the token (not exactly sure what that is - maybe the exp?). I'll need to look into how these tokens are being used.